pokete_classes/poke.py

```diff
@@ -281,6 +281,33 @@ def from_dict(cls, _dict):
                    _dict.get("attacks", None), _dict.get("effects", []),
                    shiny=_dict.get("shiny", False), nature=_dict.get("nature"))
 
+    @classmethod
+    def wild(cls, poke, _xp):
+        """Simulates learning attacks for wild poketes
+        ARGS:
+            poke: The poketes descriptor
+            _xp: The poketes given xp"""
+        obj = cls(poke, _xp)
+        for i in range(obj.lvl()):
+            if (
+                i % 5 == 0
+                and (new_attack := LearnAttack.get_attack(obj)) is not None
+            ):
+                obj.attacks.append(new_attack)
+            else:
+                break
```

Comment:
This should not work, as the poke only learns the attack of `i = 0`. The else will break the loop immediately at `i = 1`. If you want the pokete to learn as many attacks, as it has levels (divided by 5), this break should be removed:

```suggestion
```

Reply:
Oh yes, I forgot about the `i % 5 == 0` condition.